### PR TITLE
#107

### PR DESCRIPTION
**Pull request by paulinatar**

### CODE REVIEW

File: src/components/Dashboard/Form/DocumentForm.scss [hunk 1]
```diff
@@ -5,13 +5,20 @@
...
 
 div .buttons {
   @include flex(space-between flex-end);
   padding: 30px 90px;
+}
+.border{
+  border: 1px solid red;
+  width: 600px;
+
 }
```
Comment (magul) [inline src/components/Dashboard/Form/DocumentForm.scss]: add a newlline at the end

File: src/components/Dashboard/Form/FieldWrapper/FieldWrapper.scss [hunk 2]
```diff
@@ -31,6 +31,7 @@
 
 .Input {
   width: 100%;
+  //width: 50%;
```
Comment (magul) [inline src/components/Dashboard/Form/FieldWrapper/FieldWrapper.scss]: remove commented code

File: package-lock.json [hunk 3]
```diff
@@ -14760,9 +14760,9 @@
       }
     },
     "tree-kill": {
-      "version": "1.2.1",
-      "resolved": "https://registry.npmjs.org/tree-kill/-/tree-kill-1.2.1.tgz",
-      "integrity": "sha512-4hjqbObwlh2dLyW4tcz0Ymw0ggoaVDMveUB9w8kFSQScdRLo0gxO9J7WFcUBo+W3C1TLdFIEwNOWebgZZ0RH9Q==",
+      "version": "1.2.2",
```
Comment (magul) [inline package-lock.json]: I'm fine with that update, but that's strange, that without change in `package.json` it ugraded one package.

File: src/components/Dashboard/Form/Applicants/Applicant.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import FieldWrapper from "../FieldWrapper/FieldWrapper";
+import styles from "./Applicant.scss";
+
+class Applicant extends Component {
+  state = {
+    isVisible: false
+  };
+  handleOnButton = () => {
```
Comment (Rafalkufel) [inline src/components/Dashboard/Form/Applicants/Applicant.js]: Maybe:
`handleOnButton = () => {
this.setState({ isVisible: !this.state.isVisible })
}`

File: src/components/Dashboard/Form/Applicants/Applicant.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  render() {
+    return (
+      <div className={styles.Applicant}>
+        <button
+          onClick={this.handleOnButton}
+          type="button"
+          className={styles.developedDiv}
+        >
+          {" "}
+          +{" "}
```
Comment (Rafalkufel) [inline src/components/Dashboard/Form/Applicants/Applicant.js]: What is this for?

File: src/components/Dashboard/Form/Applicants/Applicant.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        </button>
+        <label className={styles.labelForAdd}>
```
Comment (Rafalkufel) [inline src/components/Dashboard/Form/Applicants/Applicant.js]: Maybe `labelForAddApplicantButton`?

File: src/components/Dashboard/Form/Applicants/Applicant.js [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    }
+  };
+  render() {
+    return (
+      <div className={styles.Applicant}>
+        <button
+          onClick={this.handleOnButton}
+          type="button"
+          className={styles.developedDiv}
+        >
```
Comment (Maciek246) [inline src/components/Dashboard/Form/Applicants/Applicant.js]: Maybe a better solution is using existing component [Button](https://github.com/CodeForPoznan/alinka-electron/blob/master/src/components/Button/Button.js)

File: src/components/Dashboard/Form/Applicants/Applicant.js (hunk 4 above)
Comment (Maciek246) [inline src/components/Dashboard/Form/Applicants/Applicant.js]: Maybe
```suggestion
  handleButtonClick = () => {
```

File: src/appContent.js [hunk 8]
```diff
@@ -14,11 +14,17 @@ const fieldLabels = {
...
+    city1: "Miasto",
+    city2: "Miasto",
+    lastName1: "Nazwisko wnioskodawcy",
+    name2: "Imię drugiego wnioskodawcy",
+    lastName2: "Nazwisko drugiego wnioskodawcy",
+    postalCode1: "Kod pocztowy",
+    postalCode2: "Kod pocztowy",
+    street1: "Ulica i numer domu",
+    street2: "Ulica i numer domu",
+    //houseNumber: "Numer lokalu/mieszkania",
```
Comment (Maciek246) [inline src/appContent.js]: Please remove unnecessary comment

File: src/components/Dashboard/Form/Applicants/Applicant.js [hunk 9]
```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import FieldWrapper from "../FieldWrapper/FieldWrapper";
+import styles from "./Applicant.scss";
+
+class Applicant extends Component {
+  state = {
+    isVisible: false
+  };
+  handleOnButton = () => {
+    if (this.state.isVisible === false) {
```
Comment (damianwojtkowski) [inline src/components/Dashboard/Form/Applicants/Applicant.js]: I think we can use here one line if/else statement.

Comment (paulinatar) [reply src/components/Dashboard/Form/Applicants/Applicant.js]: done

File: src/appContent.js (hunk 8 above)
Comment (paulinatar) [reply src/appContent.js]: done

File: src/components/Dashboard/Form/Applicants/Applicant.js (hunk 4 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/Applicants/Applicant.js]: done

File: src/components/Dashboard/Form/Applicants/Applicant.js (hunk 6 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/Applicants/Applicant.js]: Done

File: src/components/Dashboard/Form/Applicants/Applicant.js (hunk 5 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/Applicants/Applicant.js]: ESLint added this, but I corrected , as ESlint allows

File: src/components/Dashboard/Form/Applicants/Applicant.js (hunk 4 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/Applicants/Applicant.js]: yes :) done

File: src/components/Dashboard/Form/FieldWrapper/FieldWrapper.scss (hunk 2 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/FieldWrapper/FieldWrapper.scss]: done

File: src/components/Dashboard/Form/DocumentForm.scss (hunk 1 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/DocumentForm.scss]: done

File: src/components/Dashboard/Form/Applicants/Applicant.js (hunk 7 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/Applicants/Applicant.js]: yes, but I need different styles, and I don't know is possible overwrite styles.Button, and don't know how. Any idea ?

Comment (Maciek246) [reply src/components/Dashboard/Form/Applicants/Applicant.js]: Should we have a button with different styles than others in the application?  If so, maybe change in Button component  - reading styles from props and if there are not given, using default styles

File: package-lock.json (hunk 3 above)
Comment (Bosmanfrx) [reply package-lock.json]: Probably just `npm install`. Without additional flag it updates package-lock automatically

Comment (magul) [reply package-lock.json]: but if `package-lock.json` is present and there's no change in `package.json` (which is a case here), there shouldn't be updates in `package-lock.json` (beyond widely known problem with overwriting `http` with `https` and vice versa).

But that's still a theoretical problem, as I'm fine with this update here.

File: src/components/Dashboard/Form/Applicants/Applicant.scss [hunk 10]
```diff
@@ -0,0 +1,20 @@
+@import '../../../../scss/_mixins.scss';
+.Applicant{
+  @include flex(wrap);
+  width: 650px ;
+}
+button.developedDiv {
```
Comment (agatakozinska) [inline src/components/Dashboard/Form/Applicants/Applicant.scss]: use only class ;) don't mix that with tag, css modules give us the unique classes anyway

File: src/components/Dashboard/Form/Applicants/Applicant.scss [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+  border-radius: 50%;
+  color: $white;
+  cursor: pointer;
+  font-size: $font-lg;
+  font-weight: $bold;
+  margin: 30px 0;
+  outline: none;
+  padding: 0 9px ;
+}
+label.labelForAdd {
```
Comment (agatakozinska) [inline src/components/Dashboard/Form/Applicants/Applicant.scss]: the same -> use only class ;) don't mix that with tag, css modules give us the unique classes anyway

File: src/components/Dashboard/Form/Applicants/Applicant.scss [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+button.developedDiv {
+  background-color: $orange;
+  border: none;
+  border-radius: 50%;
+  color: $white;
+  cursor: pointer;
+  font-size: $font-lg;
+  font-weight: $bold;
+  margin: 30px 0;
+  outline: none;
```
Comment (agatakozinska) [inline src/components/Dashboard/Form/Applicants/Applicant.scss]: just a tip for the future, outline is good for accessibility, while setting it to none you remove it, then a solution is quite easy, you can add `outline-color` and set it to transparent also add `outline-style` and set it to none

File: src/components/Dashboard/Form/DocumentForm.scss [hunk 13]
```diff
@@ -5,13 +5,20 @@
...
   padding-left: 90px;
+  padding-right: 120px;
 }
 
 div .buttons {
   @include flex(space-between flex-end);
   padding: 30px 90px;
+}
+.border{
+  border: 1px solid red;
```
Comment (agatakozinska) [inline src/components/Dashboard/Form/DocumentForm.scss]: I see `red` ;> no variable?

File: src/components/Dashboard/Form/Applicants/Applicant.scss (hunk 10 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/Applicants/Applicant.scss]: done

File: src/components/Dashboard/Form/Applicants/Applicant.scss (hunk 11 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/Applicants/Applicant.scss]: done

File: src/components/Dashboard/Form/Applicants/Applicant.scss (hunk 12 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/Applicants/Applicant.scss]: thank you :) done

File: src/components/Dashboard/Form/DocumentForm.scss (hunk 13 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/DocumentForm.scss]: unused class/trash

File: src/components/Dashboard/Form/Applicants/Applicant.js (hunk 7 above)
Comment (paulinatar) [reply src/components/Dashboard/Form/Applicants/Applicant.js]: I will consult with @agatakozinska  how many different buttons we will need and if there will be more of them I will add ticket for that and change Button component

File: src/components/Dashboard/Form/DocumentForm.scss [hunk 14]
```diff
@@ -17,8 +17,7 @@ div .buttons {
   @include flex(space-between flex-end);
   padding: 30px 90px;
 }
-.border{
-  border: 1px solid red;
-  width: 600px;
-
-}
\ No newline at end of file
+//.border{
```
Comment (Rafalkufel) [inline src/components/Dashboard/Form/DocumentForm.scss:20]: Do you really need that commented fragment?

File: src/components/Dashboard/Form/Applicants/Applicant.js [hunk 15]
```diff
@@ -6,25 +6,21 @@ class Applicant extends Component {
...
+          onClick={this.handleButtonClick}
           type="button"
           className={styles.developedDiv}
         >
-          {" "}
-          +{" "}
+          {" + "}
         </button>
-        <label className={styles.labelForAdd}>
+        <label className={styles.labelForAddApplicantButton}>
```
Comment (akozinska) [inline src/components/Dashboard/Form/Applicants/Applicant.js:23]: usually if element is not inside label tag, label itself should have an attribute `for` (in Reach `htmlFor`) and a element `id`

File: src/components/Dashboard/Form/DocumentForm.scss [hunk 16]
```diff
@@ -17,8 +17,7 @@ div .buttons {
...
+//  border: 1px solid red;
+//  width: 600px;
+//}
```
Comment (akozinska) [inline src/components/Dashboard/Form/DocumentForm.scss:23]: if we don't use it, you can remove it

File: src/components/Dashboard/Form/Applicants/Applicant.js [hunk 17]
```diff
@@ -6,25 +6,21 @@ class Applicant extends Component {
...
-    } else {
-      this.setState({ isVisible: false });
-    }
+  handleButtonClick = () => {
+    this.setState({ isVisible: !this.state.isVisible });
   };
   render() {
     return (
       <div className={styles.Applicant}>
+        {/*<Button onClick={this.handleButtonClick}>+</Button>*/}
```
Comment (agatakozinska) [inline src/components/Dashboard/Form/Applicants/Applicant.js:15]: if it is not needed, you can remove it

File: src/components/Dashboard/Form/Applicants/Applicant.js [hunk 18]
```diff
@@ -6,25 +6,21 @@ class Applicant extends Component {
...
         <button
-          onClick={this.handleOnButton}
+          onClick={this.handleButtonClick}
           type="button"
           className={styles.developedDiv}
         >
-          {" "}
-          +{" "}
+          {" + "}
```
Comment (agatakozinska) [inline src/components/Dashboard/Form/Applicants/Applicant.js:21]: is it eslint fix? looks weird :D maybe its better to add `// eslint-disable-next-line` ?

File: src/components/Dashboard/Form/Applicants/Applicant.js (hunk 15 above)
Comment (agatakozinska) [inline src/components/Dashboard/Form/Applicants/Applicant.js:23]: usually if element is not inside label tag, label itself should have an attribute `for` (in Reach `htmlFor`) and a element `id`

File: src/components/Dashboard/Form/DocumentForm.scss (hunk 16 above)
Comment (agatakozinska) [inline src/components/Dashboard/Form/DocumentForm.scss:23]: if we don't use it, you can remove it

**Pull request merged**